src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CallbackValidations.java

```diff
@@ -192,9 +192,40 @@ static boolean hasOcr(CaseDetails theCase) {
             .map(validation -> validation
                 .mapError(throwable -> "Invalid journeyClassification. Error: " + throwable.getMessage())
             )
+            .map(validation -> validateClassification(validation, theCase))
```

Comment:
How about doing this mapping inside the previous map call (before `mapError`)? Then `validateClassification` would have to worry about the result of previous validation steps and could just accept classification and case.

Reply:
good call by doubting this orchestration. let's see, we could do something like following:

```java
map(validation -> validation
    .mapError(throwable -> "...")
    .flatMap(classification -> validateClassification(theCase, classification))
)
```

solves the nesting of validation objects used in `validateClassification`.